### PR TITLE
fix(wizard): adjust padding, var names

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/1501

### CODE REVIEW

File: src/patternfly/components/Wizard/wizard.scss [hunk 1]
```diff
@@ -422,12 +421,13 @@
 }
 
 .pf-c-wizard__main {
+  padding: var(--pf-c-wizard__main--PaddingTop) var(--pf-c-wizard__main--PaddingRight) var(--pf-c-wizard__main--PaddingLeft) var(--pf-c-wizard__main--PaddingRight);
```
Comment (christiemolloy) [inline src/patternfly/components/Wizard/wizard.scss]: Should the last two variables be "PaddingBottom" and "PaddingLeft"

**Comment by mcoker** [comment]
@christiemolloy doh! thanks, updated. It is technically a breaking change, but the component was just added this week, I wonder if it would be OK. @jschuler is working on the pf-react component, and I would like to see these changes there. @dgutride what do you think?

**Comment by dgutride** [comment]
I'd say it's still ok @mcoker - nobody is using it yet

**Comment by srambach** [comment]
LGTM.

**Pull request merged**